src/lexer.rs

```diff
@@ -11,18 +11,27 @@ pub struct Lexer<'a> {
 	indentation: usize,
 	target_indent: usize,
 	new_line: bool,
+	peeked: Option<Token>,
```

Comment:
Rename `peeked` to `peek`.

Reply:
Done. I was following the naming convention of `Peekable` but I prefer `peek`.